geometry_msgs/Pose2D task_goal
---
int16 robot_no